src/writing-policies/spec/06-signature-verifier-policies.md

```diff
@@ -0,0 +1,147 @@
+# Signature verifier policies
+
+Kubewarden implements support for the [Sigstore](https://www.sigstore.dev/)
+standard. This allows to implement a Secure Supply Chain for your cluster.
+
+Part of that is to ensure that all container images run the in the cluster are
+signed and verified, proving that they come from their stated authors, and
+haven't been tampered with (for other concerns, see the docs on [how we implement a Secure Supply Chain for the policies themselves](TODO link to https://github.com/kubewarden/docs/pull/100/files)).
+
+Sigstore signatures of containers are stored as layers of the container image.
+Policies that want to check Sigstore signatures of containers need then to check
+those container layers, and would need to pull the signature layers to see the
+signatures themselves.
+
+Obtaining and operating with those container layers needs to happen outside of
+the waPC guest (the policy), and on the waPC host (Kubewarden's
+`policy-server`).
+
+Just like for context-aware policies, the guest-host intercommunication is
+performed using the regular waPC host calling mechanism, and any guest
+implementing the waPC intercommunication mechanism is able to request this
+information.
+
+The different language SDKs for Kubewarden policies take care of all that, and
+provide functions for container image verification. This ensures a safe and
+tested codepath for verification. In addition, pulling images and
+cryptographically verifying signatures can be time and computationally
+expensive, so the SDKs stack ensures that both the image pulls and the
+verification computations are cached.
```

Comment:
Again, I'm trying to not make it about "just container images":

```suggestion
tested codepath for verification. In addition, pulling data from a registry and
cryptographically verifying signatures can be time and computationally
expensive, so the WebAssembly runtime (PolicyServer, `kwctl`) ensures both
signature pulls and verification computations are cached.
The cached entries are automatically expired after 60 seconds to prevent stale data from being served.
```